tests/integration/test_plugins.py

```diff
@@ -15,14 +16,20 @@
 
 from .constants import ALLOWED_PLUGINS, INSTALLED_PLUGINS, REMOVED_PLUGINS
 from .helpers import (
+    create_kubernetes_cloud,
+    create_secret_file_credentials,
     gen_git_test_job_xml,
     gen_test_job_xml,
+    gen_test_pipeline_with_custom_script_xml,
     get_job_invoked_unit,
     install_plugins,
+    kubernetes_test_pipeline_script,
     wait_for,
 )
 from .types_ import KeycloakOIDCMetadata, LDAPSettings, UnitWebClient
 
+logger = logging.getLogger()
```

Comment:
Perhaps getLogger(__name__) to show which module is logging would help :D

Reply:
I'll put this in the follow-up PR as well.